ACM_General/home/views.py

```diff
@@ -8,19 +8,38 @@
 
 
 def index(request):
-    # Grabs the 3 nearest upcoming events that haven't expired yet
+    """
+        Grabs the next 3 events that aren't past expiry,
```

Comment:
No indent. And it's not guaranteed that it is three its with relation to settings.MAX_HOME_FLIER_COUNT.